lms/services/dashboard.py

```diff
@@ -259,7 +259,7 @@ def get_course_roster(
             query = self._user_service.get_users_for_course(
                 role_scope=RoleScope.COURSE,
                 role_type=RoleType.LEARNER,
-                course_id=lms_course.course.id,
+                lms_course=lms_course,
```

Comment:
We were going a long way to go from LMSCouse -> Grouping here to then go the opposite way, Grouping -> LMSCourse in the query.

We likely ended up here after a few independent refactors.